core/docker/compose.sh

```diff
@@ -1,6 +1,11 @@
 composeFilePath=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 
 if [ "$1" == "up" ]; then
+    docker-compose -f "$composeFilePath"/docker-compose-mongo.yml up -d
```

Comment:
@bradsawadye This `docker-compose-mongo.yml` needs to be referenced in the `down` and `destroy` sections of the script else it won't be cleaned up properly

Reply:
addressed in https://github.com/openhie/instant/pull/41/commits/26046074e16c00c733162602fec1a77fc888941b